docs/vm/backup-restore.md

```diff
@@ -68,3 +68,42 @@ You can choose to either delete or retain the previous volumes. By default, all
 1. Click `Create`.
 
 The restore process can be viewed from the `Virtual Machines` page.
+
+## Restore a new VM on another Harvester cluster
+
+_Available as of v1.0.0_
+
+You can migrate a backup to another Harvester cluster and create a new VM based on it.
```

Comment:
Users can now restore a new VM on another cluster by leveraging the VM metadata & content backup feature.

Reply:
Updated it. Thank you.